tests/fixtures/optimizer/optimizer.sql

```diff
@@ -1388,3 +1388,40 @@ WHERE
 ORDER BY
   COUNT(DISTINCT `cs1`.`cs_order_number`)
 LIMIT 100;
+
+# title: query that uses an UNNEST which doesn't introduce a new scope
```

Comment:
Should we be (1) including regressions as-is or (2) simplifying them down to their core issue?

(1) gives us more confidence we're fixing a specific issue
(2) might be easier to debug if it ever fails later

Reply:
Yeah, agreed that we should be simplifying these, which is why I added a minimal test in `test_optimizer.py` too. This one was more like a "bonus" end-to-end test in my view, but it may be redundant. Do you think we should just get rid of it?

Reply:
I don't have a strong opinion - just opening the discussion.

I like these string-to-string fixture tests better than the scope unit tests.

I also like the idea of a regression test suite. But I guess its _slightly_ higher on the "testing pyramid".

I would vote for a minimal string-to-string fixture test, unless the regression is a very important use case or captures some complexity that's hard to confidently simplify.

Reply:
That makes sense to me 👍

Let me see if I can simplify this one a bit more. May just move it to the projection pushdown fixture file.